submissions/sadoharu/task_js_dom_/scripts.js

```diff
@@ -0,0 +1,73 @@
+window.addEventListener("DOMContentLoaded", () => {
+    const pages = document.querySelectorAll(".section__page");
+    const navList = document.querySelector(".nav-bar__list");
+  
+    const navListItem = [
+      {
+        title: "Overview",
+        icon: "icon-browser",
+      },
+      {
+        title: "About",
+        icon: "icon-about-dot-me",
+      },
+      {
+        title: "Achievement",
+        icon: "icon-award",
+      },
+      {
+        title: "Projects",
+        icon: "icon-files-empty",
+      },
+      {
+        title: "Reviews",
+        icon: "icon-rate_review",
+      },
+    ];
+  
+    navListItem.forEach((item, i) => {
+      const liItem = document.createElement("li");
+      liItem.classList.add("nav-bar__list_item");
+      liItem.tabIndex = 0;
+      liItem.setAttribute("data-index", i);
+      liItem.innerHTML = `
+              <span class="nav-bar__list_icon ${item.icon}"></span>
+              <span class="nav-bar__list_title">${item.title}</span>
+              `;
+      navList.append(liItem);
+    });
+  
+    const togglePageVisibility = (show = 0) => {
+      pages.forEach((item) => {
+        item.classList.remove("visible");
+      });
+      pages[show].classList.add("visible");
+    };
+  
+    togglePageVisibility();
+  
+    const togglePage = (e, node) => {
+      togglePageVisibility(node.getAttribute("data-index"));
+      e.currentTarget
+        .querySelectorAll(".nav-bar__list_item")
```

Comment:
First of all, you can use `querySelector('.nav-bar__list_item .active')`. Cause you should have only one active item.

Why not save `nav-bar` into a variable and then just go thru it? Something like `navList.querySelector('.active')` looks more readable and understandable than `e.currentTarget` etc.